geode-core/src/main/java/org/apache/geode/internal/cache/ha/HARegionQueue.java

```diff
@@ -3474,12 +3478,14 @@ protected void putEventInHARegion(Conflatable event, Long position) {
       }
       // Put the reference to the HAEventWrapper instance into the
       // HA queue.
+      // logger.info("adding inputHaEventWrapper to HARegion at " + position + ":" +
+      // inputHaEventWrapper + " for " + this.regionName);
       this.region.put(position, inputHaEventWrapper);
-      // logger.info(LocalizedStrings.DEBUG, "added message at position " + position);
     } else { // (event instanceof ClientMarkerMessageImpl OR ConflatableObject OR
              // ClientInstantiatorMessage)
+      // logger.info("adding ClientUpdateMessage to HARegion at " + position + ":" + event + " for "
+      // + this.regionName);
```

Comment:
Do you want to remove this commented out info message or convert it to a debug?

Reply:
converting to debug